wp-content/themes/bpr2018/resources/assets/sass/_elements.scss

```diff
@@ -107,6 +107,42 @@
   }
 }
 
+// alert banners, alert banner button
+.alert-banner{
+  position: relative;
+}
+
+//
+@media (prefers-color-scheme: dark) {
```

Comment:
There's a lot of overlap between the stylings for dark mode and light mode; the only real difference is the color. Thus, you can write a single rule for all of the common stylings and only specify the colors in the `prefers-color-scheme` media queries.

For example:
```css
.alert-banner-close {
    position: absolute;
    top: 50%;
    ...
    background: none;
}

@media (prefers-color-scheme: dark) {
    .alert-banner-close { color: $color-grey-darkmode; }
}

@media (prefers-color-scheme: light) {
    .alert-banner-close { color: $color-grey-lightmode; }
}

```